docs/dataframe_supported.rst

```diff
@@ -0,0 +1,458 @@
+pandas.DataFrame supported APIs
+===============================
+
+The following table lists both implemented and not implemented methods. If you have need
+of an operation that is listed as not implemented, feel free to open an issue on the
+`GitHub repository`_. Contributions are also welcome!
+
++---------------------------+---------------------------------+----------------------------------------------------+
+| DataFrame method          | Modin Implementation? (Y/N/P/D) | Notes for Current implementation                   |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``T``                     | Y                               |                                                    |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``abs``                   | Y                               |                                                    |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``add``                   | Y                               | Shuffles data in operations between DataFrames     |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``add_prefix``            | Y                               |                                                    |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``add_suffix``            | Y                               |                                                    |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``agg``                   | Y                               | - Passing a dictionary for the ``func`` parameter  |
+|                           |                                 | defaults to pandas                                 |
+|                           |                                 | - Passing the string name of a numpy operation for |
+| ``aggregate``             |                                 | the ``func`` parameter defaults to pandas          |
++---------------------------+---------------------------------+----------------------------------------------------+
+| ``align``                 | D                               | Defaults to pandas                                 |
```

Comment:
May not need to explicitly mention `Defaults to pandas` in the notes if `D` represents defaults to pandas